modules/pytket-quantinuum/docs/intro.txt

```diff
@@ -119,14 +119,42 @@ The ``process_circuits`` method for the QuantinuumBackend accepts the following
 *  ``noisy_simulation`` : boolean flag to specify whether the simulator should
    perform noisy simulation with an error model (default value is ``True``).
 *  ``group`` : string identifier of a collection of jobs, can be used for usage tracking.
-*  ``max_batch_cost``: maximum HQC usable by submitted batch, default is 500.
-*  ``batch_id`` : first ``jobid`` of the batch to which this batch of circuits should be submitted. Job IDs can be retrieved from ResultHandle using ``backend.get_jobid(handle)``.
-*  ``close_batch`` : boolean flag to close the batch after the last circuit in the job (default value is ``True``).
 
 For the Quantinuum ``Backend``, ``process_circuits`` returns a ``ResultHandle`` object containing a ``job_id`` and a postprocessing ( ``ppcirc``) circuit if there is one.
 
 The ``logout()`` method clears stored JSON web tokens and the user will have to sign in again to access the Quantinuum API.
 
+Batching
+--------
+Quantinuum backends (except syntax checkers) support batching of jobs (circuits). To create
+a batch of jobs, users submit the first job, then signal that subsequent jobs should
+be added to the same batch using the handle of the first. The backend queue
+management system will start the batch as soon as the first job reaches the
+front of the queue and ensure subsequent batch jobs are run one after the other,
+until the end of the batch is reached or there are no new jobs added to the batch
+for ~1 min (at which point the batch expires and any subsequent jobs will be
```

Comment:
Does this include an automated submission after 1 minute?

Reply:
I'm not sure what you mean

Reply:
If a batch is started is that directly submitted to the API or is that done later after waiting some time or does that require any action? From my understanding of the code that is directly submitted? In the example you are using the `batch_end=True` is that required?

It might be helpful to outline to the users somewhere on witch cases it is recommended to use `start_batch` and in which the old `process_circuits` ? What do you think?

Reply:
If a batch is started it is immediately submitted to the API. The batch management system is entirely done on the backend. The 1 minute time interval is between batch job submissions to the API. 

I don't really have any solid recommendations about when to use what, which is why I've gone with just stating the system as clearly as possible and letting the user decide.

Reply:
Okay, thank you!